src/style/spacings.js

```diff
@@ -20,12 +20,7 @@ class Spacings {
   }
 
   getKeysPattern() {
-    return new RegExp(
-      _.chain(this)
-        .keys()
-        .join('|')
-        .value(),
-    );
+    return /^(s[0-9]+)/;
```

Comment:
for `Spacings` we also have "semantic" names (we load them in the private code) like `page`, `innerCard`, etc.. 
so we still need to base our regex on the keys name. 
I guess you could do the same thing u did for the Typography

Reply:
Fixed, thanks